packages/react-form-renderer/src/files/use-field-api.js

```diff
@@ -170,6 +177,8 @@ const useFieldApi = ({ name, initializeOnMount, component, render, validate, ...
     ...(arrayValidator ? { arrayValidator } : {}),
     input: {
       ...fieldProps.input,
+      value:
```

Comment:
What about?
`...(fieldProps.input.type === 'file' && typeof fieldProps.input.value === 'object' && {value: fieldProps.input.value.inputValue})`

Reply:
that's how I originally wrote it but linter did it this way

Reply:
:man_facepalming: